python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -44,7 +44,8 @@
 import numpy as np
 import pandas as pd
 from scipy.stats import iqr
-from typing import Dict, Sequence
+from smatch.matcher import sphdist
+from typing import Sequence
```

Comment:
Now that we're getting rid of the `Dict` import from `typing` on this ticket, we should tidy up the docstrings in a couple of places: `compute_stats()` and `_get_columns()`.

Reply:
That part of the code is slated for deprecation in RFC-1008 and so I'm not updating any of it.